### DISCUSSION TITLE
Does the installation of wireguard-manager impact my already installed setup?

### DISCUSSION THREAD

**Discussion by christofkac** [complexorganizations/wireguard-manager#291] [Q&A]
Hi,
I'm searching for an easier way to manage my small wireguard network and stumbled across this project.
Before I install wireguard-manager I would like to know if it changes anything in my current installation (which I'd like not to see happening) or if I can simply use the tool to manage my existing setup?

Thanks for any hint.
Bye
Christof

**Comment by ghost** [comment]
Hey you cannot use this tool to manage your current setup, since the config will most likely be very different and if u want to use this tool than do this.

```
rm -rf /etc/wireguard
```

And than run the tool and it will install and configure everything for you and u will never have to do that again.
